servicetest/capabilities/test_caps.py

```diff
@@ -186,5 +206,32 @@ def test_list_capabilities(self):
             caps = sc.list_capabilities()
             # All caps present in the service manifest(s) set up previously should be listed
             assert "test.cap" in caps and "test.cap.valid-dbus" in caps and "test.cap.broken-gw-config" in caps
+            # The default capabilities should not be visible in the list
+            assert "test.cap.default.sample" not in caps
+        finally:
+            sc.terminate()
+
+    def test_no_caps_gives_default_only(self):
```

Comment:
This name is off, it should be something like "test_defaults_are_applied" or something, as it doesn't test any interaction between set caps and default caps, the test result would be the same even if there was some cap set.